cassandra/README.md

```diff
@@ -0,0 +1,263 @@
+# Cassandra Helm Kubernetes Demo
+
+This demo is designed to show VxFlex OS integration with Kubernetes to support a single-instance stateful application. In this case, [Cassandra Database](https://cassandra.apache.org/).
```

Comment:
I think this should read "distributed stateful application" (instead of "single-instance"). Perhaps, something like:
> ...to support a distributed stateful application using a Kubernetes "Stateful Set" with persistent volume claims.